### PR TITLE
ref: cleanup managed credential docs

### PR DESCRIPTION
**Pull request by SzySteve**
* remove username and last_decrypted from the response
* move /:id/password to /:id/update

screen shot 2017-12-15 at 11 37 41 am
screen shot 2017-12-15 at 11 38 03 am

### CODE REVIEW

File: src/data/endpoints/managed.yaml [hunk 1]
```diff
@@ -247,22 +247,33 @@ endpoints:
...
                 }' \
                 https://$api_root/$version/managed/credentials/$credential_id
-  /managed/credentials/$id/password:
+  /managed/credentials/$id/update:
     group: Credentials
     type: resource
     authenticated: true
     description: >
-      Change the password for a credential.
+      Change the username and password for a credential.
```
Comment (Dorthu) [inline src/data/endpoints/managed.yaml]: you can do one or both, right?  Should this read "username and/or password"?

File: src/data/endpoints/managed.yaml [hunk 2]
```diff
@@ -247,22 +247,33 @@ endpoints:
...
     methods:
       POST:
         response: ManagedCredential
         description: >
-          Changes the password for a credential.
+          Changes the username and password for a credential.
```
Comment (Dorthu) [inline src/data/endpoints/managed.yaml]: same here, should this be "and/or" ?

File: src/data/endpoints/managed.yaml [hunk 3]
```diff
@@ -247,22 +247,33 @@ endpoints:
...
+        params:
+          username:
+            optional: True
+            description: New username to set for this credential.
+            value: myusername
+            type: String
+          password:
+            description: New password to set for this credential.
+            value: mypassword
+            type: String
```
Comment (Dorthu) [inline src/data/endpoints/managed.yaml]: password is required but username isn't?  That's ...weird 😕

Comment (yoshizzle) [reply src/data/endpoints/managed.yaml]: Weird, but correct.

**Pull request merged**